lib/instrumentation/redis.js

```diff
@@ -12,8 +12,23 @@ const {
 } = require('../shim/specs')
 
 module.exports = function initialize(_agent, redis, _moduleName, shim) {
+  const version = Number(shim.pkgVersion.split('.').shift())
```

Comment:
I'm not sure I understand this code.  The instrumentation worked just fine, it was just that the legacy mode wasn't using callbacks.